tests/jeos/firstrun.pm

```diff
@@ -229,6 +223,12 @@ sub run {
         wait_still_screen 5;
     }
 
+    # Skip ssh key enrollment
+    unless (is_sle || is_sle_micro('<=6.0') || is_leap) {
+        assert_screen 'jeos-ssh-enroll-or-not', 1000;
```

Comment:
Default timeout_scale is 3. Will so long timeout be needed?

Reply:
Because there is full disk encryption.

Reply:
I saw it took around 700s, you can set an integer minutes, e.g. 15 mins (900s). ;)

Reply:
1000s should be OK. I prefer to keep this.